packages/repack/src/webpack/plugins/ChunksToHermesBytecodePlugin/utils/composeSourceMaps.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
+import fs from 'fs-extra';
+import execa from 'execa';
+
+/**
+ * {@link composeSourceMaps} options.
+ */
+interface ComposeSourceMapsOptions {
+  /** Path to the React Native root directory. */
+  reactNativePath: string;
+
+  /**
+   * Path to the source map generated by webpack-bundle.
+   *
+   * This will be replaced with the composed source map.
+   */
+  packagerMapPath: string;
+
+  /** Path to React-Native package inside node_modules. */
+  compilerMapPath: string;
+}
+
+/**
+ * Composes source maps generated by webpack-bundle and Hermes.
+ *
+ * Removes original source map files.
+ */
+export const composeSourceMaps = async ({
+  reactNativePath,
+  packagerMapPath,
+  compilerMapPath,
+}: ComposeSourceMapsOptions) => {
+  const composedSourceMapPath = packagerMapPath + '.composed';
+
+  await execa('node', [
```

Comment:
There's no error handling for that exec call. We might want to handle it gracefully

Reply:
if there is an error when executing that command,  `execa` should throw and that should stop the plugin altogether. Not sure if we want to handle it any other way tbh

Reply:
What we do in RN CLI for example we like to add some extra context around the error. e.g. "Failed to compose source maps. Here you can find details: {error.toString()}". We also strip some stack traces, but that's not always great idea 😅